tests/snappi/pfcwd/files/pfcwd_multi_node_helper.py

```diff
@@ -125,6 +125,9 @@ def run_pfcwd_multi_node_test(api,
     speed_str = testbed_config.layer1[0].speed
     speed_gbps = int(speed_str.split('_')[1])
 
+    """ Retrieve platform information for DUT """
+    platform = duthost.facts['platform']
```

Comment:
please get the asic_type instead of platform. All mellanox devices have this behavior